src/clj/web/lobby.clj

```diff
@@ -419,7 +438,8 @@
                                                       (handle-set-last-update (:gameid lobby) uid)))
             lobby? (get-in new-app-state [:lobbies (:gameid lobby)])]
         (send-lobby-state lobby?)
-        (broadcast-lobby-list)
+        ;;(broadcast-lobby-list) - not sure why we need to broadcast this to players outside
```

Comment:
we display the chosen ids on the lobby page. this should be a bespoke message tho, not a full send

Reply:
hmm - that's not until the game has actually started though, and that's covered by starting the game, right?
(it also breaks currently if somebody leaves and rejoins)

Reply:
hah that's a good point.